modules/nw-networkpolicy-about.adoc

```diff
@@ -11,9 +11,8 @@ the OpenShift SDN plug-in for the Kubernetes Container Network Interface (CNI).
 
 [NOTE]
 ====
-The `v1` NetworkPolicy features are available only in {product-title}. This
-means that egress policy types, IPBlock, and combining `podSelector` and
-`namespaceSelector` are not available in {product-title}.
+The `v1` NetworkPolicy features are available in {product-title} except for
```

Comment:
What v1 network policies? I assume it's the Kubernetes policies, but it's not clear. Please be more explicit.